artemis-console-extension/artemis-extension/app/package.json

```diff
@@ -10,41 +10,45 @@
     "replace-version": "replace __PACKAGE_VERSION_PLACEHOLDER__ $npm_package_version ./build/static/js -r --include=\"*.js\""
   },
   "dependencies": {
-    "@hawtio/react": "1.9.4",
-    "@patternfly/react-charts": "~7.4.9",
-    "@patternfly/react-code-editor": "~5.4.18",
-    "@patternfly/react-core": "~5.4.14",
-    "@patternfly/react-icons": "~5.4.2",
-    "@patternfly/react-styles": "~5.4.1",
-    "@patternfly/react-table": "~5.4.16",
-    "@patternfly/react-tokens": "~5.4.1",
-    "@patternfly/react-topology": "~5.4.1",
-    "artemis-console-plugin": "1.2.1",
+    "@hawtio/react": "^1.10.1",
+    "@monaco-editor/react": "^4.6.0",
+    "@patternfly/react-charts": "^7.4.9",
+    "@patternfly/react-code-editor": "^5.4.18",
+    "@patternfly/react-core": "^5.4.14",
+    "@patternfly/react-icons": "^5.4.2",
+    "@patternfly/react-styles": "^5.4.1",
+    "@patternfly/react-table": "^5.4.16",
+    "@patternfly/react-tokens": "^5.4.1",
+    "@patternfly/react-topology": "^5.4.1",
+    "artemis-console-plugin": "workspace:*",
```

Comment:
Isn't this against what is written in https://github.com/hawtio/hawtio-next/tree/main/app#dependencies-dev-dependencies-peer-dependencies, i.e. "... app should declare a dependency only if it is used directly in its code"?